deployments/liqo/values.yaml

```diff
@@ -172,6 +172,10 @@ gateway:
     annotations: {}
     # -- Labels for the network gateway service.
     labels: {}
+    # -- Override the IP here if service type is LoadBalancer and you want to use a specific IP address, e.g., because you want a static LB.
+    loadBalancerIP: ""
+    # -- If service type is LoadBalancer and want to expose the gateway ports on NodePorts set it to true.
+    allocateLoadBalancerNodePorts: ""
```

Comment:
@Sharathmk99 Sorry, not yet clear to me.
This parameter, by default, looks 'true' in Kubernetes:
`allocateLoadBalancerNodePorts defines if NodePorts will be automatically
 allocated for services with type LoadBalancer.  Default is "true". It may be
 set to "false" if the cluster load-balancer does not rely on NodePorts.  If
 the caller requests specific NodePorts (by specifying a value), those
 requests will be respected, regardless of this field. This field may only be
 set for services with type LoadBalancer and will be cleared if the type is
 changed to any other type.`
 However, looking at your text, it seems to me that the default value is 'false' (you say 'set to true if you want to do something'.
 Am I missing anything here?

Reply:
You are right. I’ll update the text describing default is true, set it to false if node port is not required. Thanks.